nixos/tests/paperless.nix

```diff
@@ -13,17 +13,21 @@ import ./make-test.nix ({ lib, ... } : {
   };
 
   testScript = ''
-    $machine->waitForUnit("paperless-consumer.service");
+    machine.wait_for_unit("paperless-consumer.service")
     # Create test doc
-    $machine->succeed('convert -size 400x40 xc:white -font "DejaVu-Sans" -pointsize 20 -fill black \
-      -annotate +5+20 "hello world 16-10-2005" /var/lib/paperless/consume/doc.png');
+    machine.succeed(
+        "convert -size 400x40 xc:white -font 'DejaVu-Sans' -pointsize 20 -fill black -annotate +5+20 'hello world 16-10-2005' /var/lib/paperless/consume/doc.png"
+    )
 
-    $machine->waitForUnit("paperless-server.service");
+    machine.wait_for_unit("paperless-server.service")
     # Wait until server accepts connections
-    $machine->waitUntilSucceeds("curl -s localhost:28981");
+    machine.wait_until_succeeds("curl -s localhost:28981")
     # Wait until document is consumed
```

Comment:
these comments are good candidates for `subtest` titles.

Reply:
Done